packages/vm-config/decentral-devnet-config.json

```diff
@@ -172,8 +172,16 @@
           "args": [
             {
               "hostConnectionId": "connection-1",
-              "controllerConnectionId": "connection-0",
-              "bondDenom": "uatom"
+              "controllerConnectionId": "connection-1",
+              "bondDenom": "uatom",
+              "bondDenomLocal": "ibc/C4CFF46FD6DE35CA4CF4CE031E643C8FDC9BA4B99AE598E9B0ED98FE3A2319F9",
+              "transferChannel": {
+                "counterpartyChannelId": "channel-1",
+                "counterpartyPortId": "transfer",
+                "sourceChannelId": "channel-1",
+                "sourcePortId": "transfer"
+              },
+              "icqEnabled": false
```

Comment:
set `true` for tests, where query responses are mocked, but won't work on a local simulated chain